src/unix/linux_like/mod.rs

```diff
@@ -1600,6 +1600,10 @@ extern "C" {
         attr: *mut pthread_mutexattr_t,
         pshared: ::c_int,
     ) -> ::c_int;
+    pub fn pthread_mutexattr_setrobust(
```

Comment:
@JohnTitor I'm uncertain if that should really be here. I followed the definition of `pthread_mutexattr_*pshared` which are split like that as well:
* `pthread_mutexattr_setpshared` in `src/unix/linux_like/mod.rs`
* `pthread_mutexattr_getpshared` in `src/unix/linux_like/linux/mod.rs`

Reply:
It should available on Android, Emscripten, and Linux, right? Then it's fine. It's up to the contributors to decide which module to place it in and we put it where we need.

Reply:
Right. It seems that Android [doesn't have](https://android.googlesource.com/platform/external/pthreads/+/refs/heads/master/pthread.h) support for those functions, Empscripten [has](https://github.com/emscripten-core/emscripten/blob/f46fbc5a1eb5dc878c3bdd5a505fc17142875961/system/lib/libc/musl/include/pthread.h). I'll will move things around and add the definitions for Emscripten as well.